src/demo/pages/PickersPage/PickersPage.tsx

```diff
@@ -8,15 +8,13 @@ import {
   ComponentPage
 } from '../../components/index';
 
-import { PeoplePickerTypesExample } from './examples/PeoplePicker.Types.Example';
```

Comment:
Maybe this should be BasePickerPage that has a bunch of examples of using the BasePicker.

If we have specific derivatives we definitely want to support, they should also be split out to their own page. It is super ambiguous to have a "Pickers" page because it's unclear what it's actually describing; the BasePicker or the specific example derivatives?